privacy-snapshot.json

```diff
@@ -46,17 +46,18 @@
   "on-ramp-content.uat-api.cx.metamask.io",
   "phishing-detection.api.cx.metamask.io",
   "portfolio.metamask.io",
+  "price-api.metamask-institutional.io",
```

Comment:
@aganglada is this new host expected? Are you guys making any calls to that API on your Snap?

Reply:
We are using MMI price api in order to get rates from solana right now, this is temporary tho as we will be using platform price api as a final solution.
https://github.com/MetaMask/snap-solana-wallet/blob/main/packages/snap/src/core/clients/price-api/price-api-client.ts